markdown/mavo-markdown.js

```diff
@@ -110,46 +110,46 @@ Mavo.Elements.register("markdown", {
 
 		// Image upload
 		env.editor.addEventListener("paste", async evt => {
-			if (env.context.mavo.uploadBackend && self.FileReader) {
+			if (this.mavo.uploadBackend && self.FileReader) {
 				// Look for the first image in the clipboard
 				const item = Array.from(evt.clipboardData.items).find(item => item.kind == "file" && item.type.indexOf("image/") === 0);
```

Comment:
This also seems like it should live in one place and not be repeated everywhere uploads via paste are allowed, but that's minor.